### PR TITLE
Fixed mesa compilation and updated.

### PR DESCRIPTION
**Pull request by JL2210**
That's about it.

### CODE REVIEW

File: packages/mesa.rb [hunk 1]
```diff
@@ -45,8 +37,8 @@ def self.build
                 --prefix=#{CREW_PREFIX} \
                 --libdir=#{CREW_LIB_PREFIX} \
                 --enable-shared-glapi \
-                --with-gallium-drivers=i915,r300,r600,radeon,radeonsi,nouveau,svga,swrast,virgl \
-                --with-dri-drivers=i915,i965,nouveau,swrast,r200,radeon \
+                --with-gallium-drivers=i915,r300,r600,radeonsi,nouveau,svga,swrast,virgl \
+                --with-dri-drivers=i915,i965,nouveau,swrast,r200 \
```
Comment (skiman6010) [inline packages/mesa.rb]: Just curious, do you know why dropping radeon fixes the package?

Comment (uberhacker) [reply packages/mesa.rb]: What happens if we just remove these lines?  Does it autodetect the current drivers?  If that is the case, then specifying drivers is not necessary.

**Comment by JL2210** [comment]
No, it fails with `'radeon' is not a valid target` with those lines.

**Comment by JL2210** [comment]
No, it fails with `'radeon' is not a valid target` with radeon. It still includes radeonsi.

File: packages/mesa.rb (hunk 1 above)
Comment (JL2210) [reply packages/mesa.rb]: No. The drivers for all Chromebooks are different. This is good.

Comment (uberhacker) [reply packages/mesa.rb]: Is this driver list exhaustive?

**Pull request merged**